nucypher/characters/base.py

```diff
@@ -138,13 +156,12 @@ def __init__(self,
         #
         # Self-Character
         #
-        if is_me is True:
 
+        if is_me is True:
```

Comment:
Do we need `is_me is True`, not just `is_me` here?

Reply:
Yeah - I think just `is_me` will suffice.

Reply:
:heavy_check_mark: